docs/materials/htc/part2-ex1-files.md

```diff
@@ -42,7 +42,7 @@ Next, look at the output that appears after the `Sandbox:` line; it is the outpu
 | `_condor_stderr`  | Saved standard error from the job           |
 | `_condor_stdout`  | Saved standard output from the job          |
 | `condor_exec.exe` | The executable, renamed from `sandbox.sh`   |
-| `tmp/`            | A directory in which to put temporary files |
+| `tmp/`, `var/tmp/`| Directories in which to put temporary files |
```

Comment:
Is `tmp` still there at the top level? If not, just drop it. If so, what is the difference? (Not that the difference needs to go into the table, I’m just asking.)

Reply:
My understanding is that it depends on the config. On CHTC, it looks like tmp and var/tmp are both there. I'm fine not mentioning it, but they're both in the sandbox

Reply:
No, leave it in.